src/app/demos/code-block/code-block-demo.component.html

```diff
@@ -0,0 +1,17 @@
+<sky-code-block
+  languageType="javascript">
+  <sky-code-block
+    languageType="javascript">
+    $(document).ready(() => {
+      console.log('jQuery is ready!');
+    });
+</sky-code-block>
+
+<sky-code-block
```

Comment:
Also, due to the auto-completion that the demo does the second demo here does not show up.

I would probably also move the `Code block with the header hidden.` to be above this second demo in a `h3` or similar and then add another header above the first block.

Reply:
Will you take one more look for us now that we've made more changes? 

The demo shows what the hidden header looks like, and I think we're okay to not add an additional demo showing a code block with the header because the Code section technically already shows a code block with a header.

Reply:
Looks a lot better